scenarios/ocd/CreateWordPressDeployment/README.md

```diff
@@ -0,0 +1,466 @@
+
+1. Add the Wordpress Bitnami Helm repository
+
+    ```bash
+    helm repo add bitnami https://charts.bitnami.com/bitnami
+    ```
+
+2. Update local Helm Chart repository cache
+
+    ```bash
+    helm repo update
+    ```
+
+3. Install Wordpress workload via Helm by running the following:
+
+    ```bash
+    helm upgrade --install --cleanup-on-fail \
+        --wait --timeout 10m0s \
+        --namespace wordpress \
+        --create-namespace \
+        --set wordpressUsername="$MY_WP_ADMIN_USER" \
+        --set wordpressPassword="$MY_WP_ADMIN_PW" \
+        --set wordpressEmail="$SSL_EMAIL_ADDRESS" \
+        --set externalDatabase.host="$MY_MYSQL_HOSTNAME" \
+        --set externalDatabase.user="$MY_MYSQL_ADMIN_USERNAME" \
+        --set externalDatabase.password="$MY_MYSQL_ADMIN_PW" \
+        --set ingress.hostname="$FQDN" \
+        --values helm-wp-aks-values.yaml \
+        wordpress bitnami/wordpress
+    ```
+
+Results:
+
+<!-- expected_similarity=0.3 -->
+```text
+Release "wordpress" does not exist. Installing it now.
+NAME: wordpress
+LAST DEPLOYED: Tue Oct 24 16:19:35 2023
+NAMESPACE: wordpress
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+CHART NAME: wordpress
+CHART VERSION: 18.0.8
+APP VERSION: 6.3.2
+
+** Please be patient while the chart is being deployed **
+
+Your WordPress site can be accessed through the following DNS name from within your cluster:
+
+    wordpress.wordpress.svc.cluster.local (port 80)
+
+To access your WordPress site from outside the cluster follow the steps below:
+
+1. Get the WordPress URL and associate WordPress hostname to your cluster external IP:
+
+    export CLUSTER_IP=$(minikube ip) # On Minikube. Use: `kubectl cluster-info` on others K8s clusters
+    echo "WordPress URL: https://mydnslabel1f2c0c.eastus.cloudapp.azure.com/"
+    echo "$CLUSTER_IP  mydnslabel1f2c0c.eastus.cloudapp.azure.com" | sudo tee -a /etc/hosts
+
+2. Open a browser and access WordPress using the obtained URL.
+
+3. Login with the following credentials below to see your blog:
+
+    echo Username: wpcliadmin
+    echo Password: $(kubectl get secret --namespace wordpress wordpress -o jsonpath="{.data.wordpress-password}" | base64 -d)
+```
+
+## Browse your AKS Deployment Secured via HTTPS
+
+Run the following command to get the HTTPS endpoint for your application:
+
+> [!NOTE]
+> It often takes 2-3 minutes for the SSL certificate to propogate and the site to be reachable via https.
+
+```bash
+runtime="5 minute"; endtime=$(date -ud "$runtime" +%s); while [[ $(date -u +%s) -le $endtime ]]; do STATUS=$(kubectl get svc --namespace=ingress-nginx ingress-nginx-controller -o jsonpath='{.status.loadBalancer.ingress[0].ip}'); echo $STATUS; if [ "$STATUS" = "$MY_STATIC_IP" ]; then break; else sleep 10; fi; done
```

Comment:
There should be a result block to ensure it passes.  If it fails, the full 5 minutes will pass, then it will continue and say that it finished successfully.